pkg/proxy/engines/proxy_request.go

```diff
@@ -139,8 +153,15 @@ func (pr *proxyRequest) Fetch() ([]byte, *http.Response, time.Duration) {
 		handlerName = pc.HandlerName
 	}
 
+	// span := trace.SpanFromContext(pr.upstreamRequest.Context())
+	// _, span := tspan.NewChildSpan(pr.Context(), rsc.Tracer, "Fetch")
+	// if span != nil {
+	// 	pr.upstreamRequest = pr.upstreamRequest.WithContext(trace.ContextWithSpan(pr.upstreamRequest.Context(), span))
+	// 	defer span.End()
+	// }
+
```

Comment:
?

Reply:
this should be unneeded so as to minimize the span count, but I kept in comments in case it needs to come back, will be scrubbed in next release.